checklists/lz_checklist.en.json

```diff
@@ -134,7 +134,7 @@
       "severity": "Medium",
       "graph": "resourcecontainers| where type == 'microsoft.resources/subscriptions'| extend ManagementGroup = tostring(tags),mgmtChain = properties.managementGroupAncestorsChain| extend Compliant =( array_length(mgmtChain) < 4 and array_length(mgmtChain) > 1)",
       "training": "https://docs.microsoft.com/learn/modules/azure-architecture-fundamentals/",
-      "link": "https://docs.microsoft.com/azure/governance/management-groups/overview"
+      "link": "https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/resource-org-management-groups"

```

Comment:
@mbrat2005 what do you think of [this](https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/resource-org-management-groups#management-group-recommendations) instead for the link?  The docs have good recommendations for resource organization design area that match our review checklist.  I think it works for the next 2 or 3 review items you proposed too:
- "Enforce a sandbox management group to allow users to immediately experiment with Azure"
- "Enforce a platform management group under the root management group to support common platform policy and Azure role assignment"
- "Enforce a dedicated connectivity subscription in the Platform management group to host an Azure Virtual WAN hub, private Domain Name System (DNS), ExpressRoute circuit, and other networking resources."